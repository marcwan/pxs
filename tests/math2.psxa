        ADDVAR          王马克
        MUL             8, 5, 王马克
        DEBUGPRINT      王马克
LABEL1:
        EXIT            0
