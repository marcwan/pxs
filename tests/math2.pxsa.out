DEBUGPRINT: 王马克 = 40.000000
